### PR TITLE
MAISTRA-2454 Add multi-cluster support to federation example

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

File: pkg/servicemesh/federation/example/config-poc/common.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  echo "namespaces (mesh1-system, mesh1-bookinfo, mesh2-system, mesh2-bookinfo),"
+  echo "so there are no conflicts."
+  exit 1
+fi
+
+oc1() {
+  if [ -n "${MESH1_KUBECONFIG}" ]; then
+    oc --kubeconfig=${MESH1_KUBECONFIG} $*
+  else
+    oc
```
Comment (jwendell) [inline pkg/servicemesh/federation/example/config-poc/common.sh]: missing $*

File: pkg/servicemesh/federation/example/config-poc/common.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  fi
+}
+
+oc2() {
+  if [ -n "${MESH2_KUBECONFIG}" ]; then
+    oc --kubeconfig=${MESH2_KUBECONFIG} $*
+  else
+    oc
```
Comment (jwendell) [inline pkg/servicemesh/federation/example/config-poc/common.sh]: ditto

**Review by jwendell — APPROVED** [review]

File: pkg/servicemesh/federation/example/config-poc/common.sh (hunk 1 above)
Comment (luksa) [reply pkg/servicemesh/federation/example/config-poc/common.sh]: thanks. fixed

**Review by rcernich — APPROVED** [review]

File: pkg/servicemesh/federation/example/config-poc/install.sh [hunk 3]
```diff
@@ -14,55 +14,141 @@
...
+sed "s:{{MESH2_CERT}}:$MESH2_CERT:g" export/configmap.yaml | oc1 apply -f -
+sed -e "s:{{MESH2_ADDRESS}}:$MESH2_ADDRESS:g" -e "s:{{MESH2_DISCOVERY_PORT}}:$MESH2_DISCOVERY_PORT:g" -e "s:{{MESH2_SERVICE_PORT}}:$MESH2_SERVICE_PORT:g" export/servicemeshpeer.yaml | oc1 apply -f -
+oc1 apply -f export/exportedserviceset.yaml
+
+log "Enabling federation for mesh2"
+sed "s:{{MESH1_CERT}}:$MESH1_CERT:g" import/configmap.yaml | oc2 apply -f -
+sed -e "s:{{MESH1_ADDRESS}}:$MESH1_ADDRESS:g" -e "s:{{MESH1_DISCOVERY_PORT}}:$MESH1_DISCOVERY_PORT:g" -e "s:{{MESH1_SERVICE_PORT}}:$MESH1_SERVICE_PORT:g" import/servicemeshpeer.yaml | oc2 apply -f -
+oc2 apply -f import/importedserviceset.yaml
+
+log "Installing bookinfo in mesh1"
```
Comment (dgn) [inline pkg/servicemesh/federation/example/config-poc/install.sh:105]: thanks for this, this annoyed me everytime about this script

**Comment by gbaufake** [comment]
🥳

**Comment by luksa** [comment]
Force pushed after rebasing (to make sure the flaky unit tests no longer fail). No actual changes to the commits.

**Pull request merged**